platform/README.md

```diff
@@ -3,9 +3,11 @@
 [![Build Status](https://dfe-ssp.visualstudio.com/s198-DfE-Benchmarking-service/_apis/build/status%2FPlatform%20CICD?branchName=main)](https://dfe-ssp.visualstudio.com/s198-DfE-Benchmarking-service/_build/latest?definitionId=2595&branchName=main)
 
 ## Getting started
+[comment]: <> (I think we need a more high level overview of what is in this directory and where to find it.)
+[comment]: <> (With respect to the connection string values, can we add in where to find these / who to contact to get them. A big issue in DfE is not knowing who has access to what, and where to get certain access critical information. Adding something in here could help aleviate that in our project.)
 
 ### Running Platform APIs
-
+[comment]: <> (Can we explain what the purpose of each of these APIs is at a high level?)
```

Comment:
This is covering specifically in the architecture documentation.  We should avoid duplication, otherwise we'll introduce inconstancy, etc